marconi-chain-index/marconi-chain-index.cabal

```diff
@@ -259,10 +259,12 @@ benchmark marconi-chain-index-bench
 test-suite marconi-chain-index-test
   import:         lang
   ghc-options:    -Wno-unused-packages
+  ghc-options:    -fobject-code
```

Comment:
Why was this needed?

And if you did need it, remove duplicate `ghc-options`?

Reply:
Forgot to remove it -- it's required when one uses `cabal repl` but not for building.